.github/workflows/benchmarks.yml

```
@@ -18,7 +18,8 @@ jobs:
       - uses: actions/checkout@v4
       
       - name: Rust toolchain
-        uses: dtolnay/rust-toolchain@stable
+        name: Install pinned Rust toolchain
```

actionlint (1.7.12)

[error] key "name" is duplicated in element of "steps" section. previously defined at line:20,col:9

(syntax-check)